x-pack/test_serverless/api_integration/test_suites/security/platform_security/authorization.ts

```diff
@@ -39,7 +39,7 @@ export default function ({ getService }: FtrProviderContext) {
           'discover',
           'discover_v2',
           'reporting',
-          'siemV2',
+          'siemV3',
```

Comment:
__note:__ I know this snapshot test is huge and it's your call after all, but it feels like it'd be safer to keep snapshots for all SIEM feature versions that we currently support: `siem`, `siemV2`, and `siemV3`. This way, it would be easier to catch cases like if we extend privileges of V3 but forget to update V2 and/or V1, or, for example, I see we added new `api:lists-*` API privileges to some sub-feature privileges, and with snapshots it'd be easier to see that V2 and V1 also grant these where necessary.

Reply:
i agree, it is nice to see the changes between versions, but... as you said, it is huuuuge. 😅

i've added `siem` and `siemV2`, because, although i'm not sure how useful it will be just because of the sheer amount of lines, still, better safe than sorry.
88d0605b7b6112e2c380721cf78eaac1044e8cd5

probably i'll need to do another role migration in the next months... i'll think about some ways to improve on this.